docs/dev/development.md

```diff
@@ -392,3 +395,4 @@ In order to properly upgrade a 3rd party component, do all of the following:
    [`utils/rpms/package_info.sh`](../../utils/rpms/package_info.sh)
 1. Make any necessary changes to `utils/rpms/<component>.sh` such as adding new
    files to various packages.
+1. Update the `utils/rpms/<component>.changelog` to reflect all changes introduced.
```

Comment:
```suggestion
1. Update the `utils/rpms/<component>.changelog` to document the change and make sure the changelog file is referenced by
the `RPM_CHANGELOG="<component>.changelog"` variable in `utils/rpms/<component>.sh`.
```

Reply:
Probably can also consolidate this with the step to update the daos.spec.  Changes to daos.spec might be unecessary in many cases.